packages/devextreme/testing/tests/DevExpress.ui.widgets/button.tests.js

```diff
@@ -390,6 +390,29 @@ QUnit.module('Button', function() {
             assert.equal(clickFired, 2, 'press space on button call click action');
         });
 
+        QUnit.test('external subscriptions fires on enter', function(assert) {
+            assert.expect(2);
+
+            let clickFired = 0;
+
+            const $element = $('#button').dxButton({
+                focusStateEnabled: true
+            });
+
+            $element.on('click', () => {
+                clickFired++;
+            });
+
+            const keyboard = keyboardMock($element);
+
+            $element.trigger('focusin');
+            keyboard.keyDown('enter');
+            assert.equal(clickFired, 1, 'press enter on button call click action');
```

Comment:
it's better always use `strictEqual`. U don't use `==` instead of a `===`, right?